src/core/public/chrome/ui/header/recent_items.scss

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+.headerRecentItemsButton {
+  margin-left: $euiSizeM;
+  margin-right: $euiSizeS;
+  min-width: $euiSizeL;
+  font-size: 0 !important;
+
+  // stylelint-disable-next-line @osd/stylelint/no_modifying_global_selectors
+  .euiHeaderSectionItemButton__content,
+  .euiButtonEmpty__text {
+    line-height: 1;
```

Comment:
curious what this does?

Reply:
To eliminate extra space taken when icons are left without a text.